WalletWasabi/Tor/Socks5/Models/Fields/OctetFields/AtypField.cs

```diff
@@ -7,44 +7,19 @@ namespace WalletWasabi.Tor.Socks5.Models.Fields.OctetFields
 {
 	public class AtypField : OctetSerializableBase
 	{
-		#region Constructors
-
-		public AtypField()
-		{
-		}
-
-		#endregion Constructors
-
-		#region Statics
-
 		// https://gitweb.torproject.org/torspec.git/tree/socks-extensions.txt
 		// IPv6 is not supported in CONNECT commands.
 
-		public static AtypField IPv4
-		{
-			get
-			{
-				var atyp = new AtypField();
-				atyp.FromHex("01");
-				return atyp;
-			}
-		}
+		public static readonly AtypField IPv4 = new AtypField(0x01);
```

Comment:
It was a property previously.

```suggestion
		public static AtypField IPv4 => new AtypField(0x01);
```

Reply:
I did that based on #4582 where similar properties were changed to readonly fields. If we should use properties then I will revert that as well.
@kiminuo What do you say?

Reply:
The idea is simply to have immutable classes which you can construct via constructor that you cannot modify later. The end goal is to remove `FromByte(s)` methods.